packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -167,10 +167,6 @@ public static void setCustomClientBuilder(CustomClientBuilder ccb) {
     customClientBuilder = ccb;
   }
 
-  public static interface CustomClientBuilder {
-    public void apply(OkHttpClient.Builder builder);
-  }
```

Comment:
For backwards compatibility, can you keep the old interface name here, but mark it as deprecated. We can then remove it in a future release

```
public static interface CustomClientBuilder extends com.facebook.react.modules.network.CustomClientBuilder {}
```